content/docs/tasks/security/mutual-tls/index.md

```diff
@@ -1,48 +1,38 @@
 ---
-title: Testing Mutual TLS
+title: Istio Mutual TLS deep dive
 description: Shows you how to verify and test Istio's automatic mutual TLS authentication.
 weight: 10
 keywords: [security,mutual-tls]
 ---
 
-Through this task, you will learn how to:
-
-* Verify the Istio mutual TLS Authentication setup
-
-* Manually test the authentication
+Through this task, you will have a closer look on Istio mutual TLS, and learn to verify mutual TLS settings. This task assumes you have
+already finished the [authentication policy](docs/tasks/security/authn-policy/) tutorial and are familiar with using authentication policy to enable mutual TLS.
```

Comment:
Link should start with /
tutorial -> task

Reply:
Done